src/rust/lib/graph-editor/src/component/node.rs

```diff
@@ -327,13 +275,14 @@ impl NodeModel {
         let size = Vector2::new(width+NODE_SHAPE_PADDING*2.0, height+NODE_SHAPE_PADDING*2.0);
         self.main_area.shape.sprite.size().set(size);
         self.drag_area.shape.sprite.size().set(size);
-        self.output_area.shape.sprite.size().set(size);
         self.main_area.mod_position(|t| t.x = width/2.0);
         self.main_area.mod_position(|t| t.y = height/2.0);
         self.drag_area.mod_position(|t| t.x = width/2.0);
         self.drag_area.mod_position(|t| t.y = height/2.0);
-        self.output_area.mod_position(|t| t.x = width/2.0);
-        self.output_area.mod_position(|t| t.y = height/2.0);
+
+        self.output_ports.frp.set_size.emit(Some(size));
```

Comment:
why Some(size) and not just size?

Reply:
Because the FRP requires `Default` to be implemented and it didn't seem useful to implement a wrapper for this.